src/DockContainerWidget.cpp

```diff
@@ -1334,6 +1498,57 @@ CDockAreaWidget* CDockContainerWidget::addDockWidget(DockWidgetArea area, CDockW
 	}
 }
 
+
+//============================================================================
+COverlayDockContainer* CDockContainerWidget::createAndInitializeDockWidgetOverlayContainer(SideTabBarArea area, CDockWidget* DockWidget)
+{
+	if (d->DockManager != DockWidget->dockManager())
+	{
+        DockWidget->setDockManager(d->DockManager); // Overlay Dock Container needs a valid dock manager
+	}
+	if (!COverlayDockContainer::areaExistsInConfig(area))
+	{
+		assert(false, "Requested area does not exist in config");

```

Comment:
User Q_ASSERT_X here instead of assert - this line does not build on Windows GCC